lib/modals/updateRecords.js

```diff
@@ -20,9 +20,9 @@ class UpdateRecords extends AbstractModal {
         'Leave completely blank to register without any data.\n\n' +
         'Examples:\n' +
         ' DS 24620 8 2 297595dc199b947aa8650923619436fbdfd99fd625195111ab4efe950900cade\n' +
-        ' NS ns1.palmreader\n' +
-        ' GLUE4 ns1.palmreader 100.200.30.40\n' +
-        ' GLUE6 ns1.palmreader 9530:f7fb:dc28:c1a3:d17f:d3f0:b875:aba8\n' +
+        ' NS ns1.palmreader.\n' +
+        ' GLUE4 ns1.palmreader. 100.200.30.40\n' +
+        ' GLUE6 ns1.palmreader. 9530:f7fb:dc28:c1a3:d17f:d3f0:b875:aba8\n' +
```

Comment:
ACK to this, its correct and if you remove the gitignore commit ill merge

Reply:
Done